### ISSUE TITLE
Steam sometimes doesn't start / games crash randomly after 40-80 minutes

### ISSUE THREAD

**Issue by SootDan** [ValveSoftware/steam-for-linux#9893]
* Steam client version (build number or date): 1689034492
* Distribution (e.g. Ubuntu): Arch
* Opted into Steam client beta?: [Yes/No] No
* Have you checked for system updates?: [Yes/No] Yes
* Steam Logs: [steam-logs.tar.gz](https://github.com/ValveSoftware/steam-for-linux/files/12136175/steam-logs.tar.gz)
* GPU: AMD RX 6800

#### Please describe your issue in as much detail as possible:
Basically, after I open Steam, it shows the 'Verifying Steam installation' screen for a fraction of a second then closes. If I click open Steam enough, eventually it will open Steam in its entirety but it is tiring to do so and feels random.
Then, when I play a game (e.g. RDR2), it will eventually crash the whole system after 40-80 minutes of play, forcing me to restart by using the power button.

#### Steps for reproducing this issue:

1. Click on Steam executable multiple times
2. Play a video game after eventually managing to open Steam
3.  Inevitable crash, pressing the power button

**Comment by kisak-valve** [comment]
Hello @SootDan, if you completely close Steam, then run `steam-runtime` from a terminal, does that influence the issue you're seeing?

Regarding Red Dead Redemption 2, that's something to note on the Proton issue tracker (https://github.com/ValveSoftware/Proton/issues/3291 for that game) and maybe also mention it to your video driver vendor if you're able to find any hint that there was a video driver issue happening there.

**Comment by SootDan** [comment]
Hi @kisak-valve, thank you for your quick response.
I have tried to run Steam from the terminal multiple times, it produces the exact same issue. Sometimes it runs from the terminal, most of the time it does not. [I added the terminal output in the following .txt if you are interested.](https://github.com/ValveSoftware/steam-for-linux/files/12136673/steam-terminal.txt)
I think the fact that games crash is more of a general issue somehow related to Steam -- I spent a few days googling and there are a few people with similar problems. According to these users has to do with the libgudev package and to downgrade from 238 to 237-2, but that did not change my situation at all.
